EKF/terrain_estimator.cpp

```diff
@@ -218,3 +221,10 @@ void Ekf::checkRangeDataContinuity()
 		_range_data_continuous = false;
 	}
 }
+
+// check that the range finder data accuracy
+void Ekf::checkRangeDataAccuracy()
+{
+	// get most recent range measurement from buffer
+	_rng_data_accuracy_good = _range_buffer.get_newest().variance <= 0.3;
```

Comment:
Seems like an arbitrary magic number.

Reply:
Yes, If you agree with this scheme, I will add it as a parameteｒ as soon.

Reply:
@mhkabir  Done d6dfcd68fe93ebda7cd6ae235cc2ef9183d1a285